wordpress-mobile/WordPress-FluxC-Android
# Issue/13328 update rewind endpoint

Parent Task - [WordPress-Android 13328](https://github.com/wordpress-mobile/WordPress-Android/issues/13328)

The PR adds support
(1) to the `RewindStatus` table  for `message` and `currentEntry` coming from the rewind status endpoint.
(2) for the "QUEUED" status in the Rewind.Status enum 

**To Test**
- Install WP Android using this PR over the current version
- Make sure the app doesn't crash upon start or when in the activity log

**Merge Instructions**
Only after [WordPress Android PR ](https://github.com/wordpress-mobile/WordPress-Android/pull/13807)has been tested
